src/components/TopList/index.tsx

```diff
@@ -11,21 +11,35 @@ interface TopListProps {
   aggregatedPoolData: LiquidityPoolDetails[];
 }
 
+const MAX_LENGTH = 5;
+
 export const TopList = ({ aggregatedPoolData }: TopListProps) => {
   const [topPools, setTopPools] = useState([] as LiquidityPoolDetails[]);
+  const LIST_LENGTH =
+    aggregatedPoolData.length >= MAX_LENGTH
+      ? MAX_LENGTH
+      : aggregatedPoolData.length;
+  const hasMultiplePools = LIST_LENGTH > 1;
+
   useEffect(() => {
     setTopPools(
-      [...aggregatedPoolData].sort((a, b) => {
-        if (a.totalValueLocked > b.totalValueLocked) {
-          return -1;
-        }
-        if (a.totalValueLocked < b.totalValueLocked) {
-          return 1;
-        }
-        return 0;
-      }),
+      [...aggregatedPoolData]
+        .sort((a, b) => {
+          if (a.totalValueLocked > b.totalValueLocked) {
+            return -1;
+          }
+          if (a.totalValueLocked < b.totalValueLocked) {
+            return 1;
+          }
+          return 0;
+        })
```

Comment:
Do you think we need to make sure we're checking numbers? Just double-checking because we had an issue with this in table sort.
Maybe we could simplify the sort check to:
```
b.totalValueLocked - a.totalValueLocked
```